cluster/common.sh

```diff
@@ -413,6 +413,14 @@ function generate-etcd-cert() {
                     "server auth",
                     "client auth"
                 ]
+            },
+            "grpcproxy": {
+                "expiry": "43800h",
```

Comment:
43800h is just a large time so it never expires?

Reply:
43800h = 5 years, which means a long enough time in this context.

I copied this value from the config in the same file.